app/src/components/inputs/inputDropdown/inputDropdown.jsx

```diff
@@ -112,12 +112,20 @@ export class InputDropdown extends Component {
     }
   };
 
+  isIncludedValue(optionValue) {
```

Comment:
Looks like these changes can bring a lot of other issues.

You can take a look at app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/projectActivityControls.jsx
We have almost the same case here.

And looks like we need Import start and Import finish messages because they are used in app/src/pages/admin/projectEventsPage/eventsGrid/eventsGrid.jsx